### ISSUE TITLE
Broken in Stackblitz with `Error: Cannot load native addon because loading addons is disabled: .../node.napi.glibc.node`

### ISSUE THREAD

**Issue by seveibar** [Level/classic-level#92]
Hi everyone,

I didn't see anything on the web about this issue, but it seems like `classic-level` doesn't work in the Stackblitz environment (many modules don't!)

What's the solution here? Would love a workaround!

![image](https://github.com/Level/classic-level/assets/1910070/73ac78e9-deee-4b69-bd9c-0996504971e2)

**Comment by vweevers** [comment]
> Currently, WebContainers can only execute languages that are natively supported on the Web, including JavaScript and WebAssembly. It is not possible to run [native addons](https://nodejs.org/api/addons.html) which are usually implemented using native languages such as C++, unless they can be compiled to WebAssembly. Therefore, loading native addons is disabled by default via [`--no-addons`](https://nodejs.org/api/cli.html#--no-addons) in WebContainers. As a result, you may encounter an error that says: `Cannot load native addon because loading addons is disabled.`
>
> The solution to this is to use an alternative to the native addon which is either fully implemented in JavaScript or can be compiled to WebAssembly.
> — https://developer.stackblitz.com/platform/webcontainers/troubleshooting-webcontainers

TLDR, it's simply not supported.

**Comment by seveibar** [comment]
Maybe this issue doesn't belong on this repo too since it's definitely not something I think `classic-levels` should add support for, but I did want to help with the SEO to direct people (if there is an alternative solution)

**Comment by mcollina** [comment]
> doesn't work in the Stackblitz environment 

Stackblitz is a **fork** of Node.js, not Node.js itself.

**Comment by seveibar** [comment]
@mcollina do you think anyone cares about using `level` in environments that don't support native/c addons? I have no idea if it's common. I can understand that "if you don't support native addons you kind of aren't NodeJS- it's an issue for stackblitz etc."

(fwiw I'm also ok w/ closing this issue/seeing if other people stumble on it over time?)

I ended up using [memory-level](https://github.com/Level/memory-level) because persistence wasn't super important for my app.

**Comment by vweevers** [comment]
I don't know if StackBlitz provides access to web APIs like IndexedDB but if they do then [`browser-level`](https://github.com/Level/browser-level) is probably the best option, if you do need persistence. An alternative is https://github.com/Level/community/issues/63.

In any case, we can't add support for StackBlitz in `classic-level` because this module is specifically for Node.js.
